CIME/get_tests.py

```diff
@@ -23,6 +23,7 @@
 #     "inherit" : (suite1, suite2, ...), # Optional. Suites to inherit tests from. Default is None. Tuple, list, or str.
 #     "time"    : "HH:MM:SS",            # Optional. Recommended upper-limit on test time.
 #     "share"   : True|False,            # Optional. If True, all tests in this suite share a build. Default is False.
+#     "perf"    : True|False,            # Optional. If True, all tests in this suite will do performance tracking. Default is False.
```

Comment:
wait every test in a suite?  We have a command line argument to create_test for that.

Reply:
@rljacob , yes:

```
The idea would be to select a few tests in the integration suite, pull them into their own suite, mark them as perf=True, then have the original suite inherit the new suite.
```

Reply:
So you would still need 2 separate blocks in tests.py but they could all be launched with one create_test command?

Reply:
@rljacob , yes. It has to be this way (for this perf=True approach) since we store properties on test suites, not individual tests. Another option would be to add a testmod that turns on performance tracking.

Reply:
Take a look at how e3sm_integration is currently assembled.  Its a combo of suites which are themselves combos of suites and each of those may have to be broken up in to perf and no-perf suites.

Reply:
Yeah, that's expected. I think only a few tests will become performance tests.